src/test/ui/hygiene/no_implicit_prelude.stderr

```diff
@@ -1,8 +1,8 @@
 error: cannot find macro `panic` in this scope
   --> $DIR/no_implicit_prelude.rs:16:9
    |
-LL |         assert_eq!(0, 0);
-   |         ^^^^^^^^^^^^^^^^^
+LL |         assert!(true);
+   |         ^^^^^^^^^^^^^^
```

Comment:
This test now fails, because `assert!(expr)` was fixed very recently on the main branch to call into `core::` directly.

The history of this test is interesting:

It was originally a `println!()` to check that on edition 2015 macros still get imported even without the prelude. Then `println!()` started calling `print!()` without hygiene, which broke this test. That was simply --blessed, such that it started to test for the failure of not finding `print!()` instead.

Then when `println!()` became hygienic and started working again, the test was not fixed and changed back, but instead changed to `assert_eq!(0, 0)`, probably because the person making the change assumed this test was supposed to test for this error. (:

So the original intention was not to check that this fails, but to check that this works. So this test should probably be blessed to restore the original intention.

Reply:
What should I do given that the test passes locally but fails on CI?

Reply:
> This test now fails, because assert!(expr) was fixed very recently on the main branch to call into core:: directly.

That sounds like you need to rebase over master.

Reply:
Thanks! You're probably right. I forgot that CI merges 'master' into the branch 🤦